<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>NetViewer</class>
 <widget class="QMainWindow" name="NetViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </active>
    <inactive>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>192</red>
      <green>192</green>
      <blue>192</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </inactive>
    <disabled>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </disabled>
   </palette>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>View Tor Network</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/vidalia_common.qrc" >:/images/16x16/applications-internet.png</iconset>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="contextMenuPolicy" >
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible" >
       <bool>false</bool>
      </property>
      <widget class="RouterListWidget" name="treeRouterList" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="statusTip" >
        <string/>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="indentation" >
        <number>0</number> 
       </property>
      </widget>
      <widget class="QSplitter" name="splitter3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>7</vsizetype>
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy" >
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible" >
        <bool>false</bool>
       </property>
       <widget class="QFrame" name="frmMap" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible" >
         <bool>false</bool>
        </property>
        <widget class="CircuitListWidget" name="treeCircuitList" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>1</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="sortingEnabled" >
          <bool>false</bool>
         </property>
        </widget>
        <widget class="RouterDescriptorView" name="textRouterInfo" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>100</horstretch>
           <verstretch>100</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>458</y>
     <width>844</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="contextMenuPolicy" >
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionRefresh" />
   <addaction name="separator" />
   <addaction name="actionZoomIn" />
   <addaction name="actionZoomOut" />
   <addaction name="actionZoomToFit" />
   <addaction name="separator" />
   <addaction name="actionHelp" />
   <addaction name="actionClose" />
  </widget>
  <action name="actionRefresh" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/view-refresh.png</iconset>
   </property>
   <property name="text" >
    <string>Refresh</string>
   </property>
   <property name="statusTip" >
    <string>Refresh the list of routers and connections</string>
   </property>
   <property name="toolTip" >
    <string>Refresh the list of routers and connections</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/help-browser.png</iconset>
   </property>
   <property name="text" >
    <string>Help</string>
   </property>
   <property name="statusTip" >
    <string>Show network map help</string>
   </property>
   <property name="toolTip" >
    <string>Show the network map help</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/emblem-unreadable.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="statusTip" >
    <string>Close the network map</string>
   </property>
   <property name="toolTip" >
    <string>Close the network map</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/zoom-in.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
   <property name="statusTip" >
    <string>Zoom in on the network map</string>
   </property>
   <property name="toolTip" >
    <string>Zoom in on the network map</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/zoom-out.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
   <property name="statusTip" >
    <string>Zoom out on the network map</string>
   </property>
   <property name="toolTip" >
    <string>Zoom out on the network map</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionZoomToFit" >
   <property name="icon" >
    <iconset resource="../res/vidalia_common.qrc" >:/images/22x22/zoom-to-fit.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom To Fit</string>
   </property>
   <property name="statusTip" >
    <string>Zooms to fit all currently displayed circuits</string>
   </property>
   <property name="toolTip" >
    <string>Zooms to fit all currently displayed circuits</string>
   </property>
   <property name="font" >
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>RouterDescriptorView</class>
   <extends>QTextEdit</extends>
   <header>gui/network/routerdescriptorview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>RouterListWidget</class>
   <extends>QTreeWidget</extends>
   <header>gui/network/routerlistwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>CircuitListWidget</class>
   <extends>QTreeWidget</extends>
   <header>gui/network/circuitlistwidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/vidalia_common.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>NetViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>403</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
